scalafix-core/shared/src/main/scala/scalafix/internal/rule/RuleCtxImpl.scala

```diff
@@ -22,14 +22,15 @@ import scalafix.util.SemanticdbIndex
 import scalafix.util.TokenList
 import org.scalameta.FileLine
 import org.scalameta.logger
+import scala.collection.immutable
```

Comment:
`scala.collection.immutable` is already in scope by default, so you can skip this import

Reply:
I added that there because I thought that the alias for Seq[A] that' in scope were actually the 

```
scala.collection.Seq[A]
```

and that would allow both mutable and immutable tokens sequence. If that's not the case I'm happy to remove it. Thanks a lot for the help here.

Reply:
It's not necessary, since `scala.collection.Seq` defaults to the immutable version. ;)

https://github.com/scala/scala/blob/v2.12.1/src/library/scala/collection/Seq.scala#L37

Reply:
thnaks